src/Tests/TestApprover.cs

```diff
@@ -0,0 +1,74 @@
+﻿namespace NServiceBus.Core.Tests
```

Comment:
None of the other classes in the test project have a namespace, so this could be removed, which would let you remove the other new using statements...